flume/CHANGES_NEXT_RELEASE

```diff
@@ -5,3 +5,5 @@
 * Encourage the usage of FQDNs regarding HDFS namenodes when using Kerberos (#309)
 * Fixed the RoundRoundChannelSelector, now it works for multiple storages (#298)
 * Allow for infinite TTL (#236)
+* Added OrionTestSink (#307)
+* Fixed OrionMySQLSink logger (now it logs as OrionMySQLSink, not as OrionHDFSSink)
```

Comment:
Given you have created an issue for this, maybe is a good idea to use it in the text, i.e.:

```
Fixed OrionMySQLSink logger (now it logs as OrionMySQLSink, not as OrionHDFSSink) (#321)
```